src/utils/calculator.test.ts

```diff
@@ -0,0 +1,36 @@
+import { GatewayOracle } from 'ardrive-core-js';
+import type { ArweaveOracle } from 'ardrive-core-js/lib/arweave_oracle';
+import { expect } from 'chai';
+import { SinonSpiedInstance, spy } from 'sinon';
+import { Calculator } from './calculator';
+
+const someValidByteCount = 1000; // 1KB
+
+describe('Ar<>Data calculator', () => {
+	let spyedOracle: SinonSpiedInstance<ArweaveOracle>;
+	let calculator: Calculator;
+
+	before(() => {
+		spyedOracle = spy(new GatewayOracle());
+		calculator = new Calculator(spyedOracle);
+	});
+
+	it('Zero oracle calls when just constructed', () => {
+		expect(spyedOracle.getWinstonPriceForByteCount.callCount === 0);
```

Comment:
The syntax here is a bit off. All of these tests will pass no matter if they're wrong or not, because they're just saying `expect(true)` or `expect(false)` every time and not actually checking anything yet.

`expect(spyedOracle.getWinstonPriceForByteCount.callCount).to.equal(0)` would be the correct syntax for this one

Similarly below: `expect(estimation).to.be.undefined;`

Reply:
U right

Reply:
Fixed this